### ISSUE TITLE
Congratulated for a rank I haven't reached

### ISSUE THREAD

**Issue by ggorlen** [Codewars/codewars.com#2015]
#### Describe the bug

Upon visiting the site after not logging in for a couple weeks, a flash message telling me I'd ranked up to 2 kyu appeared:

> You have ranked up! Powerful you are. Your new overall rank is 2 kyu.

But I hadn't touched or solved any katas since my last visit and my profile correctly shows that I'm still 3 kyu / 96.7%.

#### To Reproduce

This was a one-off occurrence for me and I can't reproduce it, although my "recent alerts" dropdown still shows the incorrect message as shown in the screenshot. Text is:

> Congratulations! Your new overall rank is 2 kyu.

#### Expected Behavior

I expected to not see any congratulatory message until I had actually leveled up to 2 kyu.

#### Screenshots

![Screenshot from 2020-01-25 12-47-25__](https://user-images.githubusercontent.com/17895165/73127389-5d29d400-3f74-11ea-8586-ae79396cc2bb.png)

![Screenshot from 2020-01-25 12-57-31__](https://user-images.githubusercontent.com/17895165/73127338-826a1280-3f73-11ea-9f6b-4436f1772b36.png)

#### Contexts

- Codewars username: ggorlen
- OS: Ubuntu 18.04 bionic
- Browser: Firefox
- Browser Version: 72.0.1

**Comment by kazk** [comment]
This is most likely because you had completed a kata that was once approved as much higher rank and then later adjusted. Recently, there were few kata that was approved with much higher rank by some new user and it was unapproved to be re-ranked.

So you did rank up, but then ranked down :|

**Comment by kazk** [comment]
Yeah, you had solved [Calculate Parity bit!](https://www.codewars.com/kata/5df261342964c80028345a0a) which was once approved as `3 kyu`. That should make you rank up. Then it was adjusted to `7 kyu`. So you ranked down.

**Comment by ggorlen** [comment]
I figured it was likely something to do with that since I'm close to the cutoff. Thanks.
